pkg/services/ngalert/database_mig.go

```diff
@@ -107,3 +107,19 @@ func alertInstanceMigration(mg *migrator.Migrator) {
 	mg.AddMigration("add index in alert_instance table on def_org_id, def_uid and current_state columns", migrator.NewAddIndexMigration(alertInstance, alertInstance.Indices[0]))
 	mg.AddMigration("add index in alert_instance table on def_org_id, current_state columns", migrator.NewAddIndexMigration(alertInstance, alertInstance.Indices[1]))
 }
+
+func alertmanagerConfigurationMigration(mg *migrator.Migrator) {
+	alertConfiguration := migrator.Table{
+		Name: "alert_configuration",
```

Comment:
I think that `alert_configuration` can be confusing. I would prefer `alertmanager_configuration` or just `alertmanager`.

Reply:
The thing is that this might not be alertmanager specific, I have the feeling we might end up including things that are not only related to the alertmanager but how we configure the "alert service" as a whole.

Reply:
I don't have a strong opinion, whatever works for me.